zookeeper/include/zk_nn_client.h

```diff
@@ -33,8 +35,12 @@ typedef struct {
   // java.lang.String)
   std::uint64_t access_time;
   std::uint64_t modification_time;
-  char owner[256];  // the client who created the file
-  char group[256];
+  char owner[MAX_USERNAME_LEN];  // the client who created the file
+  char group[MAX_USERNAME_LEN];
+  // TODO(danielbalkum): justify max of 20 readers on a file. Higher? Lower?
```

Comment:
we can probably get away with less for now but
i'd say just decide now and remove this todo.

Reply:
@danielbalkum